lib/mapper.js

```diff
@@ -52,6 +49,7 @@ exports.page = function(page, settings){
 
 exports.screen = function(screen, settings){
   var result = createCommonGAForm(screen, settings);
```

Comment:
What were the results of your test? That's something we should probably make note of. Did it prevent the call from going through, or taint the data? Seems like it's only necessary to remove it if it actually prevents the call from going through, that's what I'm most curious about.

Assuming we do need to remove it: If we need to delete data inside of `#screen` then the page data is by definition not common to all calls. We should move the pageview stuff into a separate helper and call it where appropriate.